### PR TITLE
added mcq

### PR DESCRIPTION
**Pull request by DavidHaighton**
added way to edit mcqs

### CODE REVIEW

File: src/main/java/com/based/basedsurvey/BasedSurveyApplication.java [hunk 1]
```diff
@@ -10,4 +18,33 @@ public static void main(String[] args) {
         SpringApplication.run(BasedSurveyApplication.class, args);
     }
 
+    @Bean
```
Comment (mack-wallace) [inline src/main/java/com/based/basedsurvey/BasedSurveyApplication.java:21]: remove

File: src/main/java/com/based/basedsurvey/controller/WebQuestionController.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+     * @param question the question
+     */
+    private void fillModel(Model model, Question question){
+        model.addAttribute("survey", question.getSurvey());
+        model.addAttribute("question", question);
+        List<String> options = new ArrayList<>();
+        String type = "";
+        if(question instanceof MultiplechoiceQuestion){
+            options = ((MultiplechoiceQuestion)question).getOptions();
+            type = "MQC";
```
Comment (mack-wallace) [inline src/main/java/com/based/basedsurvey/controller/WebQuestionController.java:108]: replace "MQC" with a constant.

File: src/main/resources/templates/EditQuestion.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <title>Based Survey</title>
+
+  <style>
+    table, th, td {
+      border: 1px solid;
+    }
+  </style>
+</head>
+<body>
+  <h1 th:text="'Question for Survey with ID:'+${survey.getId()}">Survey Info</h1>
```
Comment (mack-wallace) [inline src/main/resources/templates/EditQuestion.html:14]: optional, but it would be nice if this was a link to go back to my survey edit part

**Review by mack-wallace — CHANGES_REQUESTED** [review]
left some review comments

File: src/main/java/com/based/basedsurvey/BasedSurveyApplication.java (hunk 1 above)
Comment (DavidHaighton) [reply src/main/java/com/based/basedsurvey/BasedSurveyApplication.java:21]: oops!

File: src/main/resources/templates/EditQuestion.html (hunk 3 above)
Comment (DavidHaighton) [reply src/main/resources/templates/EditQuestion.html:14]: Good idea for the alpha release

**Comment by mack-wallace** [comment]
this needs unit tests

**Review by mack-wallace — APPROVED** [review]
lgtm

**Pull request merged**